tailscale.sh

```diff
@@ -38,53 +38,36 @@ tar xzf tailscale.tgz
 tar_dir="$(echo ${tarball} | cut -d. -f1-3)"
 test -d $tar_dir
 
-# create our target directory structure
-mkdir -p tailscale/usr/{bin,sbin,lib/{systemd/system,extension-release.d}}
+# Create binaries directory in home
+mkdir -p /home/deck/.bin
 
-# pull things into the right place in the target dir structure
-cp -rf $tar_dir/tailscale tailscale/usr/bin/tailscale
-cp -rf $tar_dir/tailscaled tailscale/usr/sbin/tailscaled
+# pull binaries
+cp -rf $tar_dir/tailscale /home/deck/.bin/tailscale
+cp -rf $tar_dir/tailscaled /home/deck/.bin/tailscaled
 
-# write a systemd extension-release file
-echo -e "ID=steamos\nVERSION_ID=${VERSION_ID}" >> tailscale/usr/lib/extension-release.d/extension-release.tailscale
+# add binaries to path via bashrc if not already there
+if [ $(cat /home/deck/.bashrc | grep -c 'export PATH="/home/deck/.bin:$PATH"') -eq 0 ]; then
+  echo 'export PATH="/home/deck/.bin:$PATH"' >> /home/deck/.bashrc
+fi
 
-# create the system extension folder if it doesn't already exist, remove the old version of our tailscale extension, and install our new one
-mkdir -p /var/lib/extensions
-rm -rf /var/lib/extensions/tailscale
```

Comment:
I'd like us to think about cleaning up past installs:
1. I would suggest we keep this line to delete the extension if the users had it already
2. I suspect we'll need to preface this deletion with `systemd-sysext unmerge && systemctl tailscaled stop`, and follow it with `systemd-sysext merge`, to ensure sysext-based Tailscale is deleted before we continue. Otherwise we may have issues if we install a systemd unit file that references new paths but the old version of the service is still running

Thoughts?

Reply:
Oh that's a really good call out. I was thinking about also writing an uninstall script. I could write that and make it more robust by also stopping and uninstalling legacy installations if they exist, and then we could loop that in. What do you think about that?

Reply:
I think I prefer the idea of a single file that does everything, as it's easier for users to just download one script that will uninstall past versions if necessary.

But what we can do is separate the uninstall commands into a separate function, to make the code easier to read.

Thoughts?

Reply:
Yeah true. I was thinking it may be useful to separate the logic so that other tools (such as the decky plugin) can loop into either the install or uninstall logic easily. I don't know if that tool will actually end up pulling this repo and calling these scripts, but that would definitely be something to think about if it does. We don't need to do that now though, we can split it up later if it actually proves necessary/useful!

Reply:
Looks like we've addressed this, @legowerewolf please resolve this conversation (according to [this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request), only folks with write access to the repo can resolve PR conversations)